elyra/templates/components/generic_properties_template.jinja2

```diff
@@ -35,16 +35,28 @@
             "items": []
           }
         },
-        "cpu": {
+        "cpu_request": {
           "type": "integer",
-          "title": "CPU",
-          "description": "For CPU-intensive workloads, you can choose more than 1 CPU (e.g. 1.5).",
+          "title": "CPU request",
+          "description": "For CPU-intensive workloads, you can request more than 1 CPU (e.g. 1.5, this is optional).",
```

Comment:
In thinking about introducing millicores, I was surprised to see the example in the comment regarding 1.5 CPUs.  Since this is an integer field, do fractional entries even work?   Seems like this field should be a `number`.

Reply:
I left the example as it was but I just tested it in my environment and the pipeline was triggered fine, it just automatically changes it from 1.5 to 1500m
<img width="203" alt="Screenshot 2023-12-21 at 19 02 39" src="https://github.com/elyra-ai/elyra/assets/82956400/49cf1a35-92d4-4206-80dc-ede771bf4d24">

Reply:
yeah, in future, we might think about leaving out the minimum 1 validation though, as there is no reason to request at least 1 CPU or 1000 millicores. Same for validating at least 1 Gig of memory for a task. 
But that is a story for later. I wonder, too, why 1.5 or any decimal value is allowed there, given it is an integer field. Stuff for later.